Fix additional compiler warnings

Fixes
../../base/files/file_util_linux.cc:33:10: error: case value evaluates to -1859950530, which cannot be narrowed to type 'unsigned long' [-Wc++11-narrowing]                                                         case static_cast<int>(BTRFS_SUPER_MAGIC):
         ^

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>

---
 base/files/file_util_linux.cc | 8 ++++----
 base/system/sys_info_posix.cc | 4 ++--
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/base/files/file_util_linux.cc b/base/files/file_util_linux.cc
index b123dd2a3d..e7df24159b 100644
--- a/base/files/file_util_linux.cc
+++ b/base/files/file_util_linux.cc
@@ -30,7 +30,7 @@ bool GetFileSystemType(const FilePath& path, FileSystemType* type) {
     case EXT2_SUPER_MAGIC:  // Also ext3 and ext4
     case MSDOS_SUPER_MAGIC:
     case REISERFS_SUPER_MAGIC:
-    case static_cast<int>(BTRFS_SUPER_MAGIC):
+    case static_cast<unsigned int>(BTRFS_SUPER_MAGIC):
     case 0x5346544E:  // NTFS
     case 0x58465342:  // XFS
     case 0x3153464A:  // JFS
@@ -40,14 +40,14 @@ bool GetFileSystemType(const FilePath& path, FileSystemType* type) {
       *type = FILE_SYSTEM_NFS;
       break;
     case SMB_SUPER_MAGIC:
-    case static_cast<int>(0xFF534D42):  // CIFS
+    case static_cast<unsigned int>(0xFF534D42):  // CIFS
       *type = FILE_SYSTEM_SMB;
       break;
     case CODA_SUPER_MAGIC:
       *type = FILE_SYSTEM_CODA;
       break;
-    case static_cast<int>(HUGETLBFS_MAGIC):
-    case static_cast<int>(RAMFS_MAGIC):
+    case static_cast<unsigned int>(HUGETLBFS_MAGIC):
+    case static_cast<unsigned int>(RAMFS_MAGIC):
     case TMPFS_MAGIC:
       *type = FILE_SYSTEM_MEMORY;
       break;
diff --git a/base/system/sys_info_posix.cc b/base/system/sys_info_posix.cc
index 7495c4c585..d6153f2c54 100644
--- a/base/system/sys_info_posix.cc
+++ b/base/system/sys_info_posix.cc
@@ -101,8 +101,8 @@ bool IsStatsZeroIfUnlimited(const base::FilePath& path) {
 
   switch (stats.f_type) {
     case TMPFS_MAGIC:
-    case static_cast<int>(HUGETLBFS_MAGIC):
-    case static_cast<int>(RAMFS_MAGIC):
+    case static_cast<unsigned int>(HUGETLBFS_MAGIC):
+    case static_cast<unsigned int>(RAMFS_MAGIC):
       return true;
   }
   return false;
